src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -461,6 +461,17 @@ template <int N_BATCH_LOOPS>
 bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ExecutionContext& ctx,
                                                      const ProblemDescription& problem) const
 {
+
+#ifdef WORKAROUND_SWDEV_503936
```

Comment:
```suggestion
#if WORKAROUND_SWDEV_503936
```
`#ifdef` is always evaluated to TRUE, even if WORKAROUND_SWDEV_503936 is 0

Reply:
yes, its typo, good catch